docs/self-hosted/install/storage.md

```diff
@@ -2,20 +2,43 @@
 # Workspace Storage
 
-Gitpod uses bucket storage to persist the contents of workspaces. Each workspace is tarballed into a single archive file which is then uploaded to the bucket.
+Gitpod uses bucket storage to persist the contents of workspaces. Each workspace is tarballed into a single archive file which is then uploaded to a separate bucket.
+
+By default Gitpod installs [MinIO](https://min.io/) as built-in bucket storage which uses a [persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to store workspace content.
+
+For more complex use case we recommend configuring more permanent means of persistence by either:
+ * Configure the contained MinIO-instance to serve as a [gateway](https://github.com/minio/minio/tree/master/docs/gateway) OR configure one of a [wide range of storage backends](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes).
+ * Bring your own storage bucket: Configure Gitpod to either connect to:
+   * your own installation of MinIO
+   * a Google Cloud Storage compatible storage solution
+
+
+## a) Configure custom MinIO instance
 
-By default Gitpod ships with [MinIO](https://min.io/) as built-in bucket storage. If you operate your own MinIO instance, or have access to Google Cloud Bucket storage you can use that one. You have the following options:
+ 1. Create a file `values.storage.yaml` with this content:
+```yaml
+wsSync:
+  remoteStorage:
+    kind: minio
+    minio:
+      endpoint: minio:9000
+      accessKey: EXAMPLEvalue
+      secretKey: Someone.Should/ReallyChangeThisKey!!
+      tmpdir: /tmp
 
-* Integrated MinIO: If not disabled, Gitpod installs MinIO in Kubernetes as a dependency of Gitpod’s helm charts.
-  MinIO itself can serve as a [gateway](https://github.com/minio/minio/tree/master/docs/gateway) to other storage providers.
-  When storing the data itself, MinIO relies on a [persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) which in turn supports a [wide range of storage backends](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes).
-* Bring your own storage bucket: Gitpod can be configured to connect to your own installation of MinIO or Google Cloud Storage compatible storage solution.
+# Disable built-in minio instance
+minio:
+  enabled: false
+```
+ 2. Redeploy Gitpod using `helm upgrade --install -f values.storage.yaml gitpod .` to apply the changes
 
-This helm chart ships with a [MinIO](https://min.io/) installation for this purpose. 
-Alternatively, you can connect to your own [MinIO](https://min.io/) installation using
- - `echo values/minio.yaml >> configuration.txt`
- - in `values.minio.yaml` change the values to match your installation
+## b) Configure the built-in MinIO instance
+ 1. Consult the chart's documentation at https://helm.min.io/
+ 2. Create a file `values.storage.yaml` with this content:
+```yaml
+minio:
+  # insert custom config here
```

Comment:
What's the custom config, i.e. as a user how do I know what I need to put here?

Reply:
The docs above point them to the MinIO config, and as I assume user who get here are more advanced know what to do, they just need a starting point.
This is meant to cover the whole part of "configure MinIO to your liking" which can be anything, really.

Alternatively we look up how to configure a gateway and put that example here.

Reply:
Here we should say that this is only necessary under special circumstances, and normally is not required. 
We could even consider leaving this out, because it's a super advanced use-case and works just like any other dependent helm chart. I.e. we do nothing special here and a user who has this use-case is likely advanced enough to figure things out using the Helm docs. I fret this confuses the 98% of users who won't need this.

Reply:
Agree. Just added it because it is mentioned in the docs above (Gateway) and did not want to leave the users without a starting point (this chapter). Fine with removing it altogether.